libs/components/lookup/src/lib/modules/country-field/country-field.component.ts

```diff
@@ -313,6 +307,7 @@ export class SkyCountryFieldComponent
           this.selectedCountry = newValue;
         }
       });
+    this.#changeDetector.markForCheck();
```

Comment:
This is why the icon wasn't showing. However, we still do call `#updateInputBox` so I went ahead and added this.